scripts/bulk-password-change.js

```diff
@@ -0,0 +1,66 @@
+const rpn = require('request-promise-native');
+process.env.NODE_TLS_REJECT_UNAUTHORIZED=0;
+const minimist = require('minimist');
+
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    'h': 'help'
+  }
+});
+
+if(argv.h || !argv.newPassword || !argv.url || !argv.user || !argv.password) {
+  console.log(`Set all user passwords to the provided password.
+
+Usage:
+    ${process.argv[1]} -h | --help
+    ${process.argv[1]} --newPassword Secret_1 --url https://localhost --user medic --password adminPass
+
+Options:
+    -h --help     Show this screen.
+    --newPassword The password to set for all users
+    --url         The url for the instance being changed
+    --user        The admin user this operation is run as
+    --password    The password for the admin user 
+
+`);
+  process.exit(0);
+}
+
+const url = argv.url;
+const user = argv.user;
+const password = argv.password;
+const newPassword = argv.newPassword;
+
+const options = {
+  uri: `${url}/api/v1/users`,
+  json: true,
+  headers: {
+    'Authorization': 'Basic ' + Buffer.from(`${user}:${password}`).toString('base64')
+  }   
+};
+
+const execute = async () => {
+  let users = [];
+  try {
+    users = await rpn.get(options);
+  } catch (e) {
+    console.log('An error while getting the list of users');
+    console.log(e);
```

Comment:
to be a little less verbose and a little more helpful, maybe don't do the `console.log(e)` or change it to `if (e.cause) console.log(e.cause)`. For sure change the first line to `console.log('An error while getting the list of users - ', e.message);` so the specific error is shown right at the top.  Which ever we decide, change it here and also on line 60 & 61.

The net result would be this:

```
node ./bulk-password-change.js --url fdjsl --user fds --password fds --newPassword dfs
An error while getting the list of users - Error: Invalid URI "fdjsl/api/v1/users"
```
Instead of this:

```
node ./bulk-password-change.js --url fdjsl --user fds --password fds --newPassword dfs
An error while getting the list of users                                                                                                    
{ RequestError: Error: Invalid URI "fdjsl/api/v1/users"                                                                                     
    at new RequestError (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request-promise-core/lib/errors.js:14:15)
    at Request.plumbing.callback (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request-promise-core/lib/plumbing.js:87:29)     
    at Request.RP$callback [as _callback] (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request-promise-core/lib/plumbing.js:46
:31)                                                                                                                                        
    at self.callback (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:185:22)  
    at Request.emit (events.js:198:13)                                                                                                      
    at Request.init (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:273:17)                                   
    at Request.RP$initInterceptor [as init] (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request-promise-core/configure/reques
t2.js:45:29)                                                                                                                                
    at new Request (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:127:8)  
    at request (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/index.js:53:10)     
    at Function.get (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/index.js:61:12)          
  name: 'RequestError',                                                                                                                       message: 'Error: Invalid URI "fdjsl/api/v1/users"',                                                                                       
  cause:                                                                                                                                    
   Error: Invalid URI "fdjsl/api/v1/users"                                                                                                  
       at Request.init (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:273:31)                                
       at Request.RP$initInterceptor [as init] (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request-promise-core/configure/req
uest2.js:45:29)                                                                                                                             
       at new Request (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:127:8)
       at request (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/index.js:53:10)                                        
       at Function.get (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/index.js:61:12)
       at execute (/home/mrjones/Documents/MedicMobile/cht-core/scripts/bulk-password-change.js:45:23)                                      
       at Object.<anonymous> (/home/mrjones/Documents/MedicMobile/cht-core/scripts/bulk-password-change.js:66:1)
       at Module._compile (internal/modules/cjs/loader.js:778:30)                                                                           
       at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
       at Module.load (internal/modules/cjs/loader.js:653:32),                                                                              
  error:                                                                                                                                    
   Error: Invalid URI "fdjsl/api/v1/users"                                                                                                         at Request.init (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:273:31)
       at Request.RP$initInterceptor [as init] (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request-promise-core/configure/req
uest2.js:45:29)                                                                                                                             
       at new Request (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/request.js:127:8)                                  
       at request (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/index.js:53:10)                                        
       at Function.get (/home/mrjones/Documents/MedicMobile/cht-core/node_modules/request/index.js:61:12)    
       at execute (/home/mrjones/Documents/MedicMobile/cht-core/scripts/bulk-password-change.js:45:23)
       at Object.<anonymous> (/home/mrjones/Documents/MedicMobile/cht-core/scripts/bulk-password-change.js:66:1)                            
       at Module._compile (internal/modules/cjs/loader.js:778:30)  
       at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)                                               
       at Module.load (internal/modules/cjs/loader.js:653:32),
  options:                                                                                                                                  
   { uri: 'fdjsl/api/v1/users',
     json: true,
     headers: { Authorization: 'Basic ZmRzOmZkcw==' },
     method: 'GET',
     callback: [Function: RP$callback],
     transform: undefined,
     simple: true,
     resolveWithFullResponse: false,
     transform2xxOnly: false },
  response: undefined }
```